tests/integration/bond_test.py

```diff
@@ -404,6 +409,27 @@ def test_replacing_port_keeps_mac_of_existing_bond(bond99_with_eth2, eth1_up):
     )
 
 
+@pytest.mark.skipif(
+    nm_major_minor_version() < 1.30,
+    reason="On NM lower than 1.30, the bond keeps the existing mac",
+)
+@pytest.mark.tier1
```

Comment:
Why is this tier 1 if Nmstate does not really care about the behavior? Do we need to keep the old test if we are fine if it changes?

Reply:
Oops. I am going to remove the tier1 mark.